src/components/card-container.js

```diff
@@ -1,24 +1,24 @@
 import { connect } from "react-redux";
+import { useState } from "react";
 import Card from "./card";
-import PlayerSelect from "./player-select";
+import FilterSelect from "./filter-select";
+
+const FilterSelectContainer = connect(state => (
+  {filterOptions: state.filterOptions}
+))(FilterSelect);
 
 const ContainerComp = (props) => {
+  const [cardInfos, setCardInfos] = useState([]);
   return (
     <div className="cardContainer">
-      {props.players.map(p => (
-        <Card key={p.key} player={p} />
+      {cardInfos.map(c => (
+        <Card key={c.name} name={c.name} filters={c.filters} removeCard={(name) => {setCardInfos(cardInfos.filter(info => (info.name !== name)))}} />
       ))}
       <div className="card">
-        <PlayerSelect />
+        <FilterSelectContainer setNewCard={(newCardInfo) => {setCardInfos([...cardInfos, newCardInfo])}} />
       </div>
     </div>
   );
 };
 
-const mapStateToProps = state => {
-  return {players: state.players};
-}
-
-const CardContainer = connect(mapStateToProps)(ContainerComp);
```

Comment:
removing this means live updates won't propagate

Reply:
wait wrong container, but I'm still leaning toward keeping the column filter definitions in redux so we can do things like save state to a cookie or url

Reply:
Fair enough. Alternatively, components that want to persist their own state can manage cookies on their own https://github.com/reactivestack/cookies/tree/master/packages/universal-cookie. URL is a good idea though too, not sure what the best option there is, I haven't looked at push state APIs in ages.